example/src/main/java/com/ncorti/slidetoact/example/SampleActivity.java

```diff
@@ -100,12 +100,22 @@ public void onSlideReset(SlideToActView view) {
             case R.id.button_bump_vibration:
                 setContentView(R.layout.content_bumb_vibration);
                 break;
+            case R.id.button_tick_custom_icon:
+                setContentView(R.layout.content_tick_custom_icon);
+                final SlideToActView tickSlider = findViewById(R.id.slide_custom_tick_icon);
+                findViewById(R.id.tick_progress).setOnClickListener(new View.OnClickListener() {
```

Comment:
I feel that the whole button/layout here is not needed.

I would just add a new button inside the `Custom Icon` screen that allows to change the complete Icon.